.yarn/patches/@semaphore-protocol-data-npm-3.10.0-48e6c0f59e.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/dist/index.mjs b/dist/index.mjs
+index 614a9093f82e9ae8e9219b65af0a8e93ea992560..f35cadc1c3623e2209c261ddde6c678442079522 100644
+--- a/dist/index.mjs
++++ b/dist/index.mjs
+@@ -1137,7 +1137,7 @@ var SemaphoreSubgraph = /** @class */ (function () {
+                                 filterFragments.push("timestamp_lte: \"".concat(jsDateToGraphqlDate(timestampLte), "\""));
+                             }
+                             if (filterFragments.length > 0) {
+-                                filtersQuery = "(where: {".concat(filterFragments.join(", "), "})");
++																filtersQuery = "(orderBy:timestamp, orderDirection:desc, where: {".concat(filterFragments.join(", "), "})");
```

Comment:
Could you remove some spaces here to read the code more clearly. To have something like:

```diff
-                                filtersQuery = "(where: {".concat(filterFragments.join(", "), "})");
+                                filtersQuery = "(orderBy:timestamp, orderDirection:desc, where: {".concat(filterFragments.join(", "), "})");
```

Like here: https://github.com/bandada-infra/bandada/issues/433#issuecomment-2082705725